resources/views/404.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts.app')
+
+@section('content')
+
+<link rel="stylesheet" href="css/404.css">
+<link href="https://fonts.googleapis.com/css?family=Poppins:400,700" rel="stylesheet">
```

Comment:
I would prefer not to introduce a new font to help keep it looking as similar to the other pages as possible.  When I removed this line and refreshed the page, it still looked pretty similar.  Do you have a compelling case for wanting to use this font?  It does look nice though, just prefer consistency.